lib/DraggableCore.es6

```diff
@@ -236,6 +239,11 @@ export default class DraggableCore extends React.Component {
   };
 
   handleDrag: EventHandler<MouseEvent> = (e) => {
+    // Stop the touch device from scrolling while user is dragging.
+    if(this.state.scrolling == true && this.state.dragging) {
```

Comment:
This could simply be `if (this.state.scrolling && this.state.dragging)` - please be aware of the existing style, spacing, etc.